workers/data_refinery_workers/processors/tximport.R

```diff
@@ -0,0 +1,63 @@
+# This script is based on the one written by J. Taroni:
+# https://github.com/jaclyn-taroni/ref-txome/blob/master/4-athaliana_tximport.R
+#
+# Two required arguments:
+# (1) --exp_dir    <experiement_dir> (directory where "quant.sf" files are found)
+# (2) --gene2txmap <path_of_gene2txmap_file>
+#
+# Two Output files:
+# (1) RDS file (saved as <experiment_dir>/txi_out.RDS)
+# (2) TPM file (saved as <experiment_dir>/gene_lengthScaledTPM.tsv)
+
+# Handle input arguments:
+library("optparse")
```

Comment:
Where possible, I think we should go the [double colon route](https://stackoverflow.com/questions/23232791/is-it-a-good-practice-to-call-functions-in-a-package-via) (e.g., `optparse::`) rather than loading/attaching packages (to avoid namespace collisions). But it's only one library, so probably not a big deal. 

Below `make_option()` would become `optparse::make_option()`, `optparse::OptionParser`, etc.